### DISCUSSION TITLE
Next JS 13 App Project is very slow to save file and showing intellisense suggestions in VS Code

### DISCUSSION THREAD

**Discussion by gidgudgod** [vercel/next.js#50417] [Help]
### Summary

I had this issue when developing a project of Next JS 13 app stable version in VS Code. The longer I run my app in dev mode the longer it takes for save, for intellicode to loads suggestions, also for prettier and eslint to auto format.

Example case:
1. I stay and edits many times in one page.tsx file, which means nextjs will compile many times as well.
2. As time goes on, the slow loading of saving and formatting comes up.
3. The slow loading comes up more clear especially after I move to another file
4. In the end I tried to restart VSCode and run dev again.
5. At beginning after restart everything goes fine, but as time goes on... back to point number 1.

This is very strange bug because only in this Next JS 13 app project that the lags happen. Meanwhile I also opened other project like React project and Next JS 13 pages project, these projects run just fine.

For now still not yet found the solution, already tried to add .next in tsconfig and disabling extension just as the github community suggested, but lag still occurs.

### Additional information

```js
Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Education
    Binaries:
      Node: 18.16.0
      npm: N/A
      Yarn: N/A
      pnpm: 8.3.0
    Relevant packages:
      next: 13.4.3
      eslint-config-next: 13.4.3
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.0.4
```

**Comment by xitanggg** [comment]
I had this issue with `prettier-plugin-tailwindcss` version `0.3`, downgrading it back to say `0.2.1` with `npm install --save-dev prettier-plugin-tailwindcss@0.2.1`  and restarting VSCode solve the issue for me

https://github.com/vercel/next.js/discussions/38477#discussioncomment-6084531
